app/browser/api/ledger.js

```diff
@@ -1482,9 +1464,10 @@ const getStateInfo = (state, parsedData) => {
     return state
   }
 
+  if (!ledgerClient) ledgerClient = require('bat-client')
   const newInfo = {
     paymentId: parsedData.properties.wallet.paymentId,
-    passphrase: parsedData.properties.wallet.keychains.passphrase,
+    passphrase: ledgerClient.prototype.getWalletPassphrase(parsedData),
```

Comment:
[minor] - it seems weird that this code accesses methods on `ledgerClient.prototype` instead of methods on an instance of the Client

Reply:
cc @mrose17

Reply:
as of https://github.com/brave-intl/bat-client/pull/13, i think you will need to construct an instance of ledgerClient in order for some stuff to work